app/presenters/publishing_api/ministers_index_presenter.rb

```diff
@@ -0,0 +1,61 @@
+module PublishingApi
+  class MinistersIndexPresenter
+    attr_accessor :item
+    attr_accessor :update_type
+
+    def initialize(item: nil, update_type: nil)
+      self.item = nil
+      self.update_type = update_type || "major"
+    end
+
+    delegate :content_id, to: :item
```

Comment:
Since `item` is `nil`, it won't have a `content_id` method. Instead we'll want to [hard code the `content_id` for this page to `324e4708-2285-40a0-b3aa-cb13af14ec5f`](https://github.com/alphagov/whitehall/blob/52aff8f61a29b3999054b5b5c94875a5534eaf9a/lib/publish_static_pages.rb#L59).

```suggestion
    def content_id
      "324e4708-2285-40a0-b3aa-cb13af14ec5f"
    end
```